libraries/SITL/SIM_TFS20L.cpp

```diff
@@ -0,0 +1,97 @@
+#include "SIM_TFS20L.h"
+#include "SITL/SIM_Aircraft.h"
+#include <AP_Math/AP_Math.h>
+#include <cmath>
+
+#if AP_SIM_TFS20L_ENABLED
+
+using namespace SITL;
+
+// Register addresses matching real sensor
+static const uint8_t REG_DIST_LOW = 0x00;
+static const uint8_t REG_DIST_HIGH = 0x01;
+static const uint8_t REG_AMP_LOW = 0x02;
+static const uint8_t REG_AMP_HIGH = 0x03;
+static const uint8_t REG_VERSION_MAJOR = 0x0C;
+
+// Limits matching real sensor
+static const uint16_t MAX_DIST_CM = 2000;  // 20m
+static const uint16_t MIN_DIST_CM = 1;     // 1cm
+
+void TFS20L::update(const class Aircraft &aircraft)  
```

Comment:
This method is called at a kHz.  You can more-closely approximate the sensor by adding a `last_updated_ms` here.